### DISCUSSION TITLE
Showing full commit message

### DISCUSSION THREAD

**Discussion by peredwardsson** [jj-vcs/jj#4364] [Q&A] [answered]
Is there a way to show the full commit message? I can't for the life of me find a good flag to show them in `jj log` or any other command.

**Comment by martinvonz** [comment, marked as answer]
A few options:

* `jj show` will include the full commit message
* Use the (apparently undocumented) `builtin_log_detailed` template: e.g. `jj log -r main -T builtin_log_detailed`
* Use a custom template: e.g. `jj log --no-graph -r main -T description`. See https://martinvonz.github.io/jj/latest/templates/ for information about what else you can pass to `-T`

**Reply by peredwardsson** [reply]
Thank you kindly!
